### ISSUE TITLE
Code Source

### ISSUE THREAD

**Issue by khaledlim** [DieBieEngineering/DieBieSlave#2]
If you can share your code source.I'm studying on a similar project
Thank you

**Comment by DieBieEngineering** [comment]
https://github.com/DieBieEngineering/DieBieMS-Firmware

**Comment by khaledlim** [comment]
Thank you so much.
if i have any question can i ask you ?

**Comment by DieBieEngineering** [comment]
Of course!

**Comment by khaledlim** [comment]
I worked a conception of  a card that contains :

-STM32f407VGT6 microcontroller.
-Ethercat Slave controller LAN9252 like you connected with SPI1 and an
EEPROM 2Mbit with I2C.
-USB 2.0 FS .
-USB 2.0 HS with external PHY USB3300
-Connector VGA.
-Connector SWD

And i'm disable to program it in Keil V5 , I already did the configuration
of pins in CubeMX.

And why and how should i use TwinCAT software

Thanks
